src/sinks/webhook/run.rs

```diff
@@ -73,7 +57,25 @@ pub(crate) fn request_loop(
 
         let body = RequestBody::from(event);
 
-        execute_fallible_request(client, url, &body, error_policy, max_retries, backoff_delay)?;
+        let result = retry::retry_operation(
+            || execute_fallible_request(client, url, &body),
+            &retry::Policy {
+                max_retries,
+                backoff_unit: backoff_delay,
+                backoff_factor: 2,
+                max_backoff: backoff_delay * 2,
```

Comment:
I'm not 100% sure what's best for these fields, this was kind of just a guess thanks to GitHub Copilot

Reply:
- `backoff_factor` is the exponent of the backoff logic (each delay will be retry_num^backoff_factor)
- `max_backoff` is the max duration of a backoff (to avoid just increasing unconstrained)